scripts/data/convert_sft_data_for_olmocore.py

```diff
@@ -10,16 +10,17 @@
         --tokenizer_name_or_path allenai/OLMo-2-1124-7B \
         --add_bos \
         --dataset_mixer_list allenai/tulu-3-sft-olmo-2-mixture-0225 1.0 \
-        --output_dir /weka/oe-adapt-default/tylerr/tulu-3-sft-olmo-2-mixture-0225-olmocore
+        --output_dir ./data/tulu-3-sft-olmo-2-mixture-0225-olmocore
```

Comment:
did you mean to change this directory to ./data/? I don't see you using the same directory anywhere else so I'm not sure